### PR TITLE
To-Do List structure

### PR DESCRIPTION
**Pull request by Marlyn-Mayienga**
In this module, we:
- Created a todo list structure using Webpack
- The tasks are added dynamically using js
-

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "build": "webpack"
```
Comment (omar-labana) [inline package.json]: - [ ] Basic commands for building Webpack in development mode, and production mode should be configured
- [ ] Kindly also setup webpack dev server to compile and serve assets

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Marlyn-Mayienga 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (Marlyn-Mayienga) [reply package.json]: @omar-labana can you give me a hint to the commands needed to be configured and the webpack dev server. This is a new territory for me so I am not sure how to implement this request

Comment (omar-labana) [reply package.json]: @Marlyn-Mayienga  for now you can just create two for one for dev and the other for building the project for production
```json
"start": "webpack serve --open",
"build": "webpack"

```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+        "start": "webpack serve --open 'google-chrome'",
```
Comment (danmainah) [inline package.json]: - [ ] Kindly ensure you setup your start command correctly , am getting this error when I run it,

image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Marlyn-Mayienga , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Github flow was correctly used

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 2 above)
Comment (Marlyn-Mayienga) [reply package.json]: @danmainah can you check it now? I corrected the google-chrome section.

**Review by elmar8287 — APPROVED** [review]
Hi, @Marlyn-Mayienga.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- Nice implementation of suggestions from the previous reviewer 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**